framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -180,6 +180,18 @@ public int runTests() throws Exception {
   	  queryMemoryUsage();
     }
 
+    // Run Apache Minio server if needed for the S3 storage
+    if(cmdParam.excludeDependencies == null || !cmdParam.excludeDependencies.contains("s3minio")) {
+      Utils.startMinio();
+    } else {
+      // Disable S3 storage plugin if Minio server is down
+      String templatePath = DrillTestDefaults.CWD + "/conf/plugin-templates/common/s3minio-storage-plugin.template";
+      boolean isSuccess = Utils.disableStoragePlugin(templatePath, "s3minio");
+      if(!isSuccess){
+        LOG.info("> Fail to disable S3 storage plugin");
```

Comment:
Update comments here and elsewhere to refer to the plugin by it's new name?

Reply:
Done.